bin/GameIndex.dbf

```diff
@@ -1165,6 +1165,12 @@ Serial = SCED-51540
 Name   = Official PlayStation 2 Magazine Demo 39
 Region = PAL-M5
 ---------------------------------------------
+Serial = SCED-51568
```

Comment:
Serial should be in numerical order.

Reply:
![image](https://user-images.githubusercontent.com/24227051/82839290-2edee580-9ecf-11ea-9045-ca95487091ee.png)
Seems to be ordered in the numerical ordering of the serials.

Reply:
It's good. My bad.